### PR TITLE
Create docker-build.yml

### PR DESCRIPTION
**Pull request by UltimateDogg**
#316 toying around with a replacement

### CODE REVIEW

**Comment by UltimateDogg** [comment]
ok have it basically working

File: .github/workflows/docker-build.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
```
Comment (ppodgorsek) [inline .github/workflows/docker-build.yml:11]: All tags should be built, would the syntax be as below?

    - *

**Review by ppodgorsek — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-build.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          - browser: "chrome"
+          - browser: "firefox"
+    env:
+      BROWSER: ${{ matrix.browser }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build image
+        run: |
+          docker build . --tag $IMAGE_NAME --file Dockerfile
```
Comment (ppodgorsek) [inline .github/workflows/docker-build.yml:38]:
```suggestion
        run: docker build . --tag $IMAGE_NAME --file Dockerfile
```

**Comment by ppodgorsek** [comment]
The Travis build file should also be removed.

File: .github/workflows/docker-build.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        include:
+          - browser: "chrome"
+          - browser: "firefox"
+    env:
+      BROWSER: ${{ matrix.browser }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build image
+        run: |
```
[the remark this answers is not in the document]
Comment (UltimateDogg) [reply .github/workflows/docker-build.yml:37]: dont know, i will try without and see

File: .github/workflows/docker-build.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        include:
+          - browser: "chrome"
+          - browser: "firefox"
+    env:
+      BROWSER: ${{ matrix.browser }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build image
+        run: docker build . --tag $IMAGE_NAME --file Dockerfile
```
Comment (ppodgorsek) [inline .github/workflows/docker-build.yml:37]: To be consistent with the lines below:

```suggestion
        run:
          docker build . --tag $IMAGE_NAME --file Dockerfile
```

**Review by ppodgorsek — APPROVED** [review]
PR reviewed and approved, the build works fine, thank you @UltimateDogg ! :)

**Pull request merged**